src/test/java/org/isf/vactype/rest/VaccineTypeControllerTest.java

```diff
@@ -88,7 +88,7 @@ void closeService() throws Exception {
 
 	@Test
 	public void testGetVaccineType_200() throws Exception {
-		String request = "/vaccinetype";
+		String request = "/vaccinetypes";
```

Comment:
again; didn't a recent change make these plural?

Reply:
Yes, but these tests were left in the singular, so I updated them.

Reply:
Yes @dbmalkovsky this change is a fix on a missing change in the previous PR, correct @npasquetto ?

Reply:
right

Reply:
it's ok for me then